### PR TITLE
Html index

### PR DESCRIPTION
**Pull request by dstorbergjr**
Added Index .HTML doc. Changed stuff.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <ul>
+      <li>Unordered</li>
+      <li>List</li>
+      <li>Example</li>
+    </ul>
+
+    <h1>Main Headline</h1>
+    <h2>Secondary Headline</h2>
+    <h3>Tertiary Headline</h3>
+    <h4>Quadiary Headline</h4>
```
Comment (voodooGQ) [inline index.html:28]: Hey Dan, can we add an <h5> tag here as well?

**Review by voodooGQ — CHANGES_REQUESTED** [review]
One small change please

File: index.html (hunk 1 above)
Comment (dstorbergjr) [reply index.html:28]: Made the change

**Review by voodooGQ — APPROVED** [review]
LGTM

**Pull request merged**